### DISCUSSION TITLE
Welcome to OpenTX-Telemetry-Widget Discussions!

### DISCUSSION THREAD

**Discussion by nm17** [iNavFlight/OpenTX-Telemetry-Widget#85] [Announcements]
## 👋 Welcome!
We’re using Discussions as a place to connect with other members of our community.

**Comment by garyleeloyd** [comment]
Hello, I'm having some trouble getting the lua telemetry working on my RadioMaster TX16S running EdgeTX 2.7.1. Is this an appropriate forum to seek help?

**Reply by stronnag** [reply]
Sure.

**Reply by garyleeloyd** [reply]
I may have figured it out. I had installed from this location: https://github.com/teckel12/LuaTelemetry/releases/tag/v1.7.2 and it was not working for me once the model was powered on (looked find UNTIL then). Once I installed from here: https://github.com/iNavFlight/OpenTX-Telemetry-Widget/releases/tag/v2.2.1 it appears to be working as expected. Thanks anyway!

**Comment by stronnag** [comment]
Yeah, one is the official INAV widget, one isn't ....
